sklearnex/linear_model/tests/test_linear.py

```diff
@@ -81,3 +82,46 @@ def test_sklearnex_import_elastic():
     assert "daal4py" in elasticnet.__module__
     assert_allclose(elasticnet.intercept_, 1.451, atol=1e-3)
     assert_allclose(elasticnet.coef_, [18.838, 64.559], atol=1e-3)
+
+
+@pytest.mark.skipif(
+    not is_dpctl_available(["gpu"]),
+    reason="gpu_blas_compute_mode is only available for select gpus",
+)
+def test_bf16_blas_epsilon():
+    from sklearnex import config_context
+    from sklearnex.linear_model import LinearRegression
+
+    X = np.array([[0.0, 1.0], [1.0, 1.0]], dtype=np.float32)
+    y = np.array([[1.0], [1.0 + np.finfo(np.float32).eps]], dtype=np.float32)
+
+    # Make changes in y not observable in bfloat16 which means casting
+    # to bf16 for gemm should yield an incorrect value, everything
+    # should be fine until the end multiplication (inverses etc.).
+
+    with config_context(
+        target_offload="gpu:0",
+        allow_fallback_to_host=False,
+        gpu_blas_compute_mode="standard",
+    ):
+        linreg_standard = LinearRegression().fit(X, y)
+
+    with config_context(
+        target_offload="gpu:0",

```

Comment:
Should we use `gpu:0` or rather just `gpu`?

Reply:
After this CI run, I will try it.